src/doc/playback_tests.cpp

```diff
@@ -464,39 +505,361 @@ TEST(Playback, PingPongWithInnerReverse)
   EXPECT_FALSE(play.isStopped());
 }
 
+// OnePingPongInsideOther series
+std::vector<int> goRight(std::vector<int> range) {
+  std::vector<int> out;
+  if (range[0] > range[1])
+    return out;
+  for (int i=range[0]; i<range[1]+1 ; ++i)
+    out.push_back(i);
+  return out;
+}
```

Comment:
No need to pass arguments as a std::vector:
```suggestion
static std::vector<int> goRight(const int a, const int b)
{
  std::vector<int> out;
  if (a > b)
    return out;
  for (int i=a; i<=b ; ++i)
    out.push_back(i);
  return out;
}
```